index.html

```diff
@@ -14,6 +15,19 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
+      <nav id="mobile-menu" class="not-active">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <ul>
+            <li><a class="mobile-link" href="#about-me">Portoflio</a></li>
+            <li><a class="mobile-link" href="##contacts">About</a></li>
```

Comment:
- You are missing some project requirements which i will highlight below
> When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- [ ] currently once a user clicks on the portfolio or about in your mobile menu it doesn't direct users to the correct part of the page. it is important that your project functions properly as this shows your attention to details which is a good quality recruiters want to see from potential hires.
- [ ] To fix this i klindly suggest that you look at the highlighted lines and make appropriate changes to the `href`.